include/tvm/relax/attrs/nn.h

```diff
@@ -197,6 +197,15 @@ struct NLLLossAttrs : public tvm::AttrsNode<NLLLossAttrs> {
   }
 };  // struct NLLLossAttrs
 
+/*! \brief Attributes used in fuse multi head attention operator */
+struct AttentionAttrs : public tvm::AttrsNode<AttentionAttrs> {
+  DataType out_dtype;
```

Comment:
probably not needed since it is not used and attention has complicated semantic than simple binary ops, `out_dtype` cannot fully express the mixed precision semantic here